TUTORIAL.md

```diff
@@ -22,15 +22,19 @@ This tutorial will be easiest if you have some knowledge of how to use Linux com
 Once those three setup items are taken care of, you're ready to start learning how to use Spotipy!
 
 ## Step 1. Creating a Spotify Account
-Spotipy relies on the Spotify API. In order to use the Spotify API, you'll need to create a Spotify developer account.
+Spotipy relies on the Spotify API. In order to use the Spotify API, you'll 
+need to create a Spotify developer account. Here is a how-to [video](https://www.youtube.com/watch?v=3RGm4jALukM) that showcases how to get 
+setup with API credentials, Redirect URIs, and Spotify Developer Accounts, all of which are necessary prerequisites to using the Spotipy library environment variables.
 
-A. Visit the [Spotify developer portal](https://developer.spotify.com/dashboard/). If you already have a Spotify account, click "Log in" and enter your username and password. Otherwise, click "Sign up" and follow the steps to create an account. After you've signed in or signed up, you should be redirected to your developer dashboard.
+A. Visit the [Spotify developer portal](https://developer.spotify.
+com/dashboard/). If you already have a Spotify account, click "Log in" and enter your username and password. Otherwise, click "Sign up" and follow the steps to create an account. After you've signed in or signed up, you should be redirected to your developer dashboard. You do not need to have a premium Spotify account, a free account will work.
 
 B. Click the "Create an App" button. Enter any name and description you'd like for your new app. Accept the terms of service and click "Create."
 
-C. In your new app's Overview screen, click the "Edit Settings" button and scroll down to "Redirect URIs." Add "http://localhost:1234" (or any other port number of your choosing). Hit the "Save" button at the bottom of the Settings panel to return to you App Overview screen.
+C. In your new app's Overview screen, click the "Edit Settings" button and scroll down to "Redirect URIs." Add "[http://localhost:1234/callback/]" (or any other port number of your choosing). Hit the "Save" button at the bottom of the Settings panel to return to you App Overview screen.
```

Comment:
The callback URL can be anything. While having `/callback` in it would work, it would also work without it.